Taskfile.yml

```diff
@@ -98,7 +98,7 @@ tasks:
       - docker-compose logs --follow
 
   env-pull:
-    desc: "Pull development environment's Docker images"
+    desc: "Pull development environment"s Docker images"
```

Comment:
Please check the diff of that PR and revert all unrelated formatting changes. This line, in particular, has invalid syntax because there are three quotation marks.